settings.gradle

```diff
@@ -270,11 +255,28 @@ include(':application-mode')
 include(':util-immutables')
 project(':util-immutables').projectDir = file('Util/util-immutables')
 
+include(':deephaven-jpy')
```

Comment:
note to other reviewers, I flattened this out since it is a net decrease in actual code, and just as clear as all the other renames we do, but without two different lambdas trying to define these projects